src/commands/force/user/create.ts

```diff
@@ -43,6 +44,26 @@ const permsetsStringToArray = (fieldsPermsets: string | string[]): string[] => {
     : fieldsPermsets.split(',').map((item) => item.replace("'", '').trim());
 };
 
+const lowercaseAllKeys = (input: Record<string, unknown>): Record<string, unknown> => {
+  return Object.keys(input).reduce(function (accum, key) {
+    accum[key.toLowerCase()] = input[key];
+    return accum;
+  }, {});
+};
+
+const standardizePasswordToBoolean = (input: unknown): boolean => {
+  if (typeof input === 'boolean') {
+    return input;
+  }
+  if (input === 'true' || input === 1) {
+    return true;
+  }
+  if (input === 'false' || input === 0) {
+    return false;
+  }
+  return true;
+};
+
```

Comment:
I think `lowercaseAllKeys` should be in kit.  Maybe it's already there, like maybe `mapKeys` would work.  If not, probably keep it here and move it later.

`standardizePasswordToBoolean` should be `asBoolean` from ts-types lib.

Reply:
I don't think asBoolean is going to handle "false" as a string correctly...it's going to check isBoolean and then use its default.

https://github.com/forcedotcom/sfdx-dev-packages/blob/02f6021a15c9937625d591feb1786bc082d8106c/packages/ts-types/src/narrowing/as.ts#L64

Reply:
`kit.mapKeys` is good.

gotta get in the habit of looking over there first.

Reply:
closest thing I see on that boolean thing is inside `Env` https://github.com/forcedotcom/sfdx-dev-packages/blob/02f6021a15c9937625d591feb1786bc082d8106c/packages/kit/src/env.ts#L209